compose/ui/ui-test-junit4/src/desktopTest/kotlin/androidx/compose/ui/test/FiltersTest.kt

```diff
@@ -72,13 +72,29 @@ class FiltersTest {
             }
         }
 
+        rule.onNodeWithTag("tag").assert(hasAnyAncestor(isDialog()))
+    }
+
+    @Test
+    fun testIsPopup() {
+        rule.setContent {
+            Popup(
+
+            ) {
+                Text(
+                    text = "Text",
+                    modifier = Modifier.testTag("tag1")
+                )
+            }
+        }
+
         rule.onNodeWithTag("tag").assert(hasAnyAncestor(isPopup()))
```

Comment:
It has `testTag("tag1")`. Does it work? why?

Reply:
Started changing it to test both `Popup` overloads, and then decided not to.